### ISSUE TITLE
Ogre Battle 64 - Person of lordly caliber

### ISSUE THREAD

**Issue by Shinriji** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#168]
# Game summary
- Game name: Ogre Battle 64 - Person of lordly caliber
- Game version: USA (1.0)
- Played status: Finished prologue and tutorial.

# Daedalus X64 summary
- Version: 0.3 nightly (May 16, 2020)

# Issues

- Sprites rendering incorrect. Can't see units in battle, so game play is affected.
- Dialog boxes are squashed, making text difficult to read.
- Game is slow, running at 17fps in story sections and 13-15 in battle.

# Screenshots
![](https://?)

# Recommended labels

- Ingame low
- Slow

**Comment by Shinriji** [comment]
With the latest update, modern renderer fixed several graphical glitches that made the game unplayable and no new glitches were introduced, so the game can change to "Playable".

**Comment by Rinnegatamante** [comment]
Not really, there still are some huge glitches eg dialog boxes and performances are poor

**Comment by Shinriji** [comment]
With 0.6 the game runs mostly fullspeed and don’t crash anymore with the groundhog day glitch. Currently on chapter 2.

**Comment by wicketTL** [comment]
@Shinriji what settings are you using in the emulator?  I tried to run it on 0.6 and am missing sprites (just black silhouettes) and the dialog boxes are scewed.

Not sure if this makes a difference, but I have NOT overclocked my vita yet.

**Comment by Rinnegatamante** [comment]
You need modern renderer.

**Comment by nokillstealing** [comment]
So is this 100% playable now? This was my fav game as a child would love to play it on the go.

**Comment by Rinnegatamante** [comment]
There still are soe graphical glitches.

**Comment by nokillstealing** [comment]
All I get is no video but with sound on modern renderer. Am I doing something wrong?

**Comment by Rinnegatamante** [comment]
Are you on nightly or v.0.6? v.0.6 should be fine.

**Comment by nokillstealing** [comment]
V0.6 I’ve messed around with the settings but I only get a black screen no graphics rendering but sound is playing. I’m on modern render

**Comment by nokillstealing** [comment]
Working phenomenal on 0.6 other than a few stray black textures and of course the squished message boxes.

**Comment by nokillstealing** [comment]
Well after the initial chapter battles are slow to unplayable. May be a memory leak somewhere.

**Comment by adrianbass360** [comment]
PLAYABLE WITHOUT ANY OF THE GLITCHES MENTIONED ABOVE, A BIT SLOW. There are very minor glitches here and there (like when doors open and between switching sceneries), and sound does lags about half the time, but the game is playable! Set sound to mono. Also use this config and this specific daedalusx64 version because using earlier versions of daedalusx64 glitch and future ones crash!

[Configs.zip](https://github.com/user-attachments/files/19820078/Configs.zip)
[DaedalusX64 v0.6b (f98727)(12-10-21).zip](https://github.com/user-attachments/files/19820077/DaedalusX64.v0.6b.f98727.12-10-21.zip)
